### PR TITLE
The Edxapp continuous delivery play.

### PR DESCRIPTION
**Pull request by macdiesel**

### CODE REVIEW

**Comment by macdiesel** [comment]
@edx/pipeline-team PTAL

**Comment by doctoryes** [comment]
How different is this pipeline from one that would install programs or insights? Looks quite similar to the programs hackathon IIRC. Is it time to make a more generic pattern *or* maybe the next iteration?

**Comment by macdiesel** [comment]
@doctoryes getting closer. I would like to pull this in and start working on something more generic. There is already a PR open for this work: https://github.com/edx/edx-gomatic/pull/24

**Comment by doctoryes** [comment]
👍

File: edxpipelines/pipelines/cd_edxapp.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    help='Save the pipeline configuration xml locally.',
+    required=False,
+    default=False
+)
+@click.option(
+    '--dry-run',
+    envvar='DRY_RUN',
+    help='Perform a dry run of the pipeline installation, and save the pre/post xml configurations locally.',
+    required=False,
+    default=False
```
Comment (feanil) [inline edxpipelines/pipelines/cd_edxapp.py]: This should have the `is_flag` option set to `True`

**Comment by feanil** [comment]
:+1:  once the `dry-run` flag is fixed.

**Comment by feanil** [comment]
:+1:

**Comment by feanil** [comment]
:+1:

**Pull request merged**